src/main/resources/META-INF/maven/site-macros.vm

```diff
@@ -48,7 +48,7 @@
 #**##end
 #end
 ##
-#macro( link $link $icon = false $omitImage = false $elemOnEmptyHref = "a" $dotSlashOnEmptyHref = false $imageClass = "imageLink" $overrideElemClass = false )
+#macro( link $link $iconClass = null $omitImage = false $elemOnEmptyHref = "a" $dotSlashOnEmptyHref = null $imageClass = "imageLink" $overrideElemClass = null )
```

Comment:
How did these changes slip into this PR? They are not related.

Reply:
This looks like a bug in GH, ignore it.